src/AppBundle/FormType/UserEditFormType.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace AppBundle\FormType;
+
+use AppBundle\Entity\User;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Aleš Kůdela
+ */
+class UserEditFormType extends AbstractType {
+
+    public function buildForm(FormBuilderInterface $builder, array $options) {
+        $builder
+                ->add("username", EmailType::class, [
+                    "label" => "E-mail",
+                    "attr" => [
+                        "class" => "form-control",
+                    ],
+                ])
+                ->add("phone", TextType::class, [
+                    "label" => "Telefon",
+                    "attr" => [
+                        "class" => "form-control",
+                    ],
+                ])
+                ->add("title", TextType::class, [
```

Comment:
Tohle by pak dávalo smysl ve chvíli kdy by adresa už byla jako samostatná entita, takhle je to pole trochu navíc :)